libvmaf/test/test_psnr.c

```diff
@@ -84,7 +85,7 @@ static char *test_16b_large_diff()
     mu_assert("wrong mse_y", almost_equal(mse_y, 4294836225.0));
     mu_assert("wrong mse_cb", almost_equal(mse_cb, 4294836225.0));
     mu_assert("wrong mse_cr", almost_equal(mse_cr, 4294836225.0));
-    
```

Comment:
In general we should avoid introducing unrelated changes into the diff, makes it much harder to read.